tests/_support/utils/router.js

```diff
@@ -0,0 +1,30 @@
+const _routes = new Map();
+
+module.exports = {
+  clearRoutes() {
+    _routes.clear();
+  },
+
+  route(routeRegEx, fixtureString) {
+    _routes.set(routeRegEx, fixtureString);
+    cy.route(...arguments);
+
+    return this;
+  },
+
+  getAPIResponse(routeString, callback) {
+    let selectedFilterString = null;
+
+    _routes.forEach(function (fixtureString, routeRegEx) {
+      if (routeRegEx.test(routeString)) {
+        selectedFilterString = /^fx:(.*)/.exec(fixtureString)[1];
```

Comment:
It's possible to also add fixtures using the `fixture:` prefix, not just `fx:`.
Let's account for that as well!